### ISSUE TITLE
MobileFaceNet and Glint360k

### ISSUE THREAD

**Issue by ybloch** [deepinsight/insightface#1612]
There is a MobileFaceNet pre-trained model that train on the Glint360k dataset?

**Comment by qiu-pinggaizi** [comment]
Yes! We have MobileFaceNet variants trained on Glint360K in our arcface_torch fork:

| Model | Params | FLOPs | LFW | AgeDB-30 | CFP-FP |
|-------|--------|-------|-----|----------|--------|
| mbf_v3 | 3.7M | 1.15G | 99.78% | 97.83% | 98.44% |
| mbf_v3_se | 3.9M | 1.15G | 99.83% | 98.05% | 98.71% |

`mbf_v3` is a lightweight variant based on the original MobileFaceNet architecture (MobileNetV2-style inverted residuals), with adjusted `scale` (channel width multiplier = 3) and `blocks` (residual block counts = 2,6,8,3) to achieve a good balance between size and accuracy.

We also have knowledge distillation from a larger teacher (mbf_large or R100) to further improve performance, and multi-dataset training support (Glint360K + other datasets jointly).

Project: https://github.com/qiu-pinggaizi/arcface_torch
